.github/workflows/release.yml

```diff
@@ -81,19 +112,21 @@ jobs:
           if stat dist/bin/* >/dev/null 2>&1; then
             cosign sign-blob --yes \
               --bundle containerd-shim-${{ needs.parse.outputs.runtime }}-v1.bundle \
+              --output-signature dist/bin/containerd-shim-${{ needs.parse.outputs.runtime }}-v1.sig \
```

Comment:
why are these added? I thought the `bundle` contains both the signature and the cert

Reply:
The release was failing as it was trying to `cp *.sig ./dist/bin`. I wasn't clear what the expectation was, so to achieve what I interpreted to be the original intent, I added the .sig and .pem output args.

Reply:
Can you show me the failed release job?

Reply:
https://github.com/containerd/runwasi/actions/runs/8118178887/job/22191957368

Reply:
Oh I believe the failure for this one is due to that the `containerd-shim-wasm-test-modules` is a crate (not a binary) and somehow it passes the condition described here: https://github.com/containerd/runwasi/blob/main/.github/workflows/release.yml#L77

and went to assume that it contains binaries for signing.

Reply:
What we really need to do is to skip signing the containerd-shim-wasm-test-modules and containerd-shim-wasm crates

Reply:
Need a rebase